### ISSUE TITLE
Incoming call notifications can't be dismissed

### ISSUE THREAD

**Issue by hrj** [LiMium/mini-vector-android#22]
When an incoming call arrives, a notification is shown that can't be dismissed.

**Comment by quazgar** [comment]
I can confirm this, the notification stays stuck.  Since by its own definition, minivector does not do voice/video chat, can incoming calls be handled without Android level notifications?

**Comment by hrj** [comment]
Agreed, this needs a fix, but I don't have the time for it. I welcome any PRs though.

(Note, since the upstream has moved to a different code base called RiotX, which hasn't stabilized yet, this repo isn't being updated often)
